go-controller/pkg/cni/cni.go

```diff
@@ -112,9 +112,11 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, clientset *ClientSet) (*Resp
 	if pr.CNIConf.DeviceID != "" {
 		var err error
 
-		netdevName, err = util.GetNetdevNameFromDeviceId(pr.CNIConf.DeviceID, pr.deviceInfo)
-		if err != nil {
-			return nil, fmt.Errorf("failed in cmdAdd while getting Netdevice name: %v", err)
+		if !pr.IsVFIO {
+			netdevName, err = util.GetNetdevNameFromDeviceId(pr.CNIConf.DeviceID, pr.deviceInfo)
+			if err != nil {
+				return nil, fmt.Errorf("failed in cmdAdd while getting Netdevice name: %v", err)
```

Comment:
nit: consider using %w instead of %v, so the error is wrapped

Reply:
fixed. Thanks @flavio-fernandes